### PR TITLE
Skios 82 feed overview page

### PR DESCRIPTION
**Pull request by jeppard**
# Skiosa - PR
- [x] added test(s), or explain why one is not needed/possible...
- [x] added comments to all the functions
- [x] added documentation to latex project
- [x] code can't be split up into smaller logical units
- [x] code is formated and linted

### CODE REVIEW

**Comment by LukasLJL** [comment]
pls fix tests

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -16,6 +17,12 @@ const routes: Routes = [
     canActivate: [LoggedInGuard],
     component: SubscriptionComponent
   },
+  {
+	path: 'feed',
+	loadChildren: () =>
+		import('./pages/feed/feed.module').then((m) => m.FeedModule),
+},
+
```
Comment (simonmorgenstern) [inline src/app/app-routing.module.ts:25]: remove module

**Comment by LukasLJL** [comment]
please redirect to something like 404, that /feed, is not allowed.

**Comment by simonmorgenstern** [comment]
> please redirect to something like 404, that /feed, is not allowed.

will be resolved when module is removed!

**Comment by simonmorgenstern** [comment]
remove either feed/feed-overview or feed-overview/feed-overview

File: src/app/core/mutations/subsscription.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+
+
+export const GENERAL_FEED_SUB_MUTATION = gql<GENERAL_FEED_SUB_MUTATION_RESPONCE, {feedId: number , isSubscribed: boolean}>`
+mutation GENERAL_FEED_SUB_MUTATION($isSubscribed: Boolean!, $feedId: Float!) {
+    changeSubscription(isSubscribed: $isSubscribed, feedId: $feedId)
+  }
+`;
+
```
Comment (simonmorgenstern) [inline src/app/core/mutations/subsscription.ts:15]: typo in filename, add comment to mutation (what it does, input, output)

File: src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<p>feed-overview-page works!</p>
+<p>nice COCK</p>
```
Comment (simonmorgenstern) [inline src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.html:2]: what the actual fuck

File: src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+@Component({
+	selector: 'app-feed-overview-page',
+	templateUrl: './feed-overview-page.component.html',
+	styleUrls: ['./feed-overview-page.component.css']
+})
+export class FeedOverviewPageComponent{
+
+
+
```
Comment (simonmorgenstern) [inline src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.ts:11]: remove whitespace

File: src/app/pages/feed/feed-overview-page/feed-overview-page.component.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.feed-overview-title {
+    margin: 0;
+    padding: 0;
+}
+
+.feed-article {
+    margin: 0.5rem;
+}
```
Comment (simonmorgenstern) [inline src/app/pages/feed/feed-overview-page/feed-overview-page.component.css:29]: use some structure like block_element_element 
feed
feed_article 
feed_overview_title

File: src/app/pages/feed/feed-overview-page/feed-overview-page.component.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
+<div class="feed-overview-container">
```
Comment (simonmorgenstern) [inline src/app/pages/feed/feed-overview-page/feed-overview-page.component.html:1]: a "div" literally is a container. feed_overview is clearly the container. we do not need unneccesary descriptions

File: src/app/pages/feed/feed-overview-page/feed-overview-page.component.ts [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+	sortArticlesOfFeed(): void {
+		this.feed.articles = this.feed.articles.sort((a, b) => {
+			return new Date(b.publishedAt).getTime() - new Date(a.publishedAt).getTime()
+		})
+	}
+
+	minNumber(a: number, b: number): number {
+		return a > b ? b : a;
+	}
+}
```
Comment (simonmorgenstern) [inline src/app/pages/feed/feed-overview-page/feed-overview-page.component.ts:91]: add comments

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (grossamos) [reply src/app/app-routing.module.ts:25]: fixed

File: src/app/core/mutations/subsscription.ts (hunk 2 above)
Comment (grossamos) [reply src/app/core/mutations/subsscription.ts:15]: fixed

File: src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.html (hunk 3 above)
Comment (grossamos) [reply src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.html]: WTF, removed

File: src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.ts (hunk 4 above)
Comment (grossamos) [reply src/app/pages/feed-overview/feed-overview-page/feed-overview-page.component.ts]: removed

File: src/app/pages/feed/feed-overview-page/feed-overview-page.component.css (hunk 5 above)
Comment (grossamos) [reply src/app/pages/feed/feed-overview-page/feed-overview-page.component.css]: fixed

File: src/app/pages/feed/feed-overview-page/feed-overview-page.component.ts (hunk 7 above)
Comment (grossamos) [reply src/app/pages/feed/feed-overview-page/feed-overview-page.component.ts]: fixed

**Comment by LukasLJL** [comment]
tests still failing

**Comment by LukasLJL** [comment]
Missing LaTeX Documentaion

**Review by LukasLJL — APPROVED** [review]
lgtm :D @simonmorgenstern pls confirm

**Review by simonmorgenstern — APPROVED** [review]
finally approved :D (need to fix my mac :P)

**Comment by grossamos** [comment]
@simonmorgenstern found a quick tutorial on how to fix your Mac https://linuxhint.com/install_linux_on_mac/

**Pull request merged**